docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -213,6 +213,22 @@ Going beyond that, you can specify an individual test method like this:
 
    $ ./runtests.py --settings=path.to.settings i18n.tests.TranslationTests.test_lazy_objects
 
+Running tests from specific point
+---------------------------------
+You can run the tests starting at the specified top-level module with
+``--start-at`` option:
+
+.. console::
+
+   $ ./runtests.py --start-at=wsgi
+
+Also, you can run the tests starting after the specified top-level module with
+``--start-after`` option:
+
+.. console::
+
+   $ ./runtests.py --start-after=wsgi
+
```

Comment:
Can we add a note about how it works with `--reverse`? i.e. that it doesn't have any impact of these options. I was confusing for me, to be honest.

Reply:
The `--reverse` option does not have any impact on these new options.
These options are for specifying the starting point of running tests. the `--reverse` option uses to reverse the order of the selected tests from the previous step.